### PR TITLE
chore: remove unnecessary deps and env variables

### PR DESCRIPTION
**Pull request by ctrlc03**
- [x] remove skip voter check as we have FreeForAllGatekeeper instead
- [x] Remove unnecessary dependencies making the build time longer

### CODE REVIEW

File: packages/interface/package.json [hunk 1]
```diff
@@ -19,31 +19,23 @@
...
     "@radix-ui/react-dialog": "^1.0.5",
     "@radix-ui/react-dropdown-menu": "^2.0.6",
     "@rainbow-me/rainbowkit": "^2.0.1",
-    "@rainbow-me/rainbowkit-siwe-next-auth": "^0.4.0",
     "@semaphore-protocol/core": "4.0.0-beta.16",
     "@semaphore-protocol/data": "4.0.0-beta.16",
     "@t3-oss/env-nextjs": "^0.8.0",
-    "@tailwindcss/forms": "^0.5.7",
     "@tanstack/react-query": "^5.24.1",
-    "@testing-library/react": "^14.1.2",
```
Comment (0xmad) [inline packages/interface/package.json:35]: Let's keep this one. It will be useful for tests.

**Review by 0xmad — APPROVED** [review]
@ctrlc03 thanks, just one minor suggestion.

File: packages/interface/package.json (hunk 1 above)
Comment (ctrlc03) [reply packages/interface/package.json:35]: is it needed as dependency or can it go in devDependencies?

Comment (0xmad) [reply packages/interface/package.json:35]: devDeps will be best

**Pull request merged**